server/service/src/sync/test/test_data/invoice.rs

```diff
@@ -422,12 +445,40 @@ fn transact_om_fields_pull_record() -> TestSyncPullRecord {
             comment: None,
             their_reference: None,
             transport_reference: Some("transport reference".to_string()),
-            created_datetime: NaiveDate::from_ymd(2022, 8, 24).and_hms(9, 33, 0),
-            allocated_datetime: Some(NaiveDate::from_ymd(2022, 8, 25).and_hms(10, 33, 0)),
-            picked_datetime: Some(NaiveDate::from_ymd(2022, 8, 26).and_hms(11, 33, 0)),
-            shipped_datetime: Some(NaiveDate::from_ymd(2022, 8, 27).and_hms(12, 33, 0)),
-            delivered_datetime: Some(NaiveDate::from_ymd(2022, 8, 28).and_hms(13, 33, 0)),
-            verified_datetime: Some(NaiveDate::from_ymd(2022, 8, 29).and_hms(14, 33, 0)),
+            created_datetime: NaiveDate::from_ymd_opt(2022, 8, 24)
+                .unwrap()
+                .and_hms_opt(9, 33, 0)
+                .unwrap(),
+            allocated_datetime: Some(
+                NaiveDate::from_ymd_opt(2022, 8, 25)
+                    .unwrap()
+                    .and_hms_opt(10, 33, 0)
+                    .unwrap(),
```

Comment:
This kind of code could be simplified but would be quite a lot of work... e.g. no need for the `Some()` wrapper